modules/mbedtls/Kconfig.tls-generic

```
@@ -505,6 +505,14 @@ config MBEDTLS_LMS
 	depends on MBEDTLS_HASH_SHA256_ENABLED
 	select PSA_WANT_ALG_SHA_256
 
+config MBEDTLS_PSA_P256M_DRIVER_ENABLED
+	bool "P256-M driver"
+	depends on MBEDTLS_PSA_CRYPTO_C
+	imply PSA_WANT_ALG_SHA_256
+	help
+	  Enable support for the optimized sofware implementation of the NIST P256
```

```suggestion
	  Enable support for the optimized sofware implementation of the NIST secp256r1
```

NIST P256 is an alias. Can be changed to secp256r1, which is the curve-type as described by the PSA crypto API spec